apps/api/src/dao/interceptors/dao-crud.interceptor.ts

```diff
@@ -6,20 +6,28 @@ import { DaoStatus } from '@sputnik-v2/dao';
 
 @Injectable()
 export class DaoCrudRequestInterceptor extends BaseCrudRequestInterceptor {
+  public static readonly defaultFields = [
+    'id',
+    'config',
+    'totalDaoFunds',
+    'numberOfMembers',
+    'numberOfGroups',
+    'activeProposalCount',
+    'totalProposalCount',
+    'accountIds',
+    'createdAt',
```

Comment:
Is 'createdAt' field added for audit purposes in our local DB? I think we're using 'blockTimestamp' to track actual blockchain timestamps.